src/analyzer.ts

```diff
@@ -94,9 +98,7 @@ export class Analyzer {
    * Note: this feature is experimental.
    */
   _fork(): Analyzer {
-    const analyzer = new Analyzer({});
```

Comment:
ugh silent build failures. Lets prioritize getting that fixed, want me to look into it or are you already?